pkg/features/kube_features.go

```diff
@@ -850,6 +845,5 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	// features that enable backwards compatibility but are scheduled to be removed
 	// ...
-	HPAScaleToZero:         {Default: false, PreRelease: featuregate.Alpha},
-	LegacyNodeRoleBehavior: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
Per feature lifecycle, shouldn't this be locked to default before removing it?

Reply:
Also, per the KEP this should be defaulted to FALSE: 

Release 4 (1.20):

"Default the legacy gate LegacyNodeRoleBehavior to off. Admins whose deployments still use the old labels may set LegacyNodeRoleBehavior=true during 1.19 to get the legacy behavior."